### PR TITLE
Run dapps tests via reusable workflow

### PR DESCRIPTION
**Pull request by lyubenkov**

### CODE REVIEW

**Comment by lyubenkov** [comment]
Green build https://github.com/neonlabsorg/proxy-model.py/actions/runs/8650943409

**Comment by lyubenkov** [comment]
@kristinaNikolaevaa please verify, in case there're no suggestions I'll change the branch in the workflow to `develop`.

File: .github/workflows/deploy.py [hunk 1]
```diff
@@ -466,37 +465,6 @@ def run_uniswap_test(project_name):
...
-    runs_before = github.get_dapps_runs_list()
-    runs_count_before = github.get_dapps_runs_count()
-    proxy_url = f"http://{proxy_ip}:9090/solana"
-    solana_url = f"http://{solana_ip}:8899/"
-    faucet_url = f"http://{proxy_ip}:3333/"
-
-    github.run_dapps_dispatches(proxy_url, solana_url, faucet_url, pr_url_for_report, full_test_suite)
-    wait_condition(lambda: github.get_dapps_runs_count() > runs_count_before, timeout_sec=180)
-
-    runs_after = github.get_dapps_runs_list()
```
Comment (kristinaNikolaevaa) [inline .github/workflows/deploy.py:487]: let's also delete all connected to old logic like "github.get_dapps_runs_list"

**Review by kristinaNikolaevaa — CHANGES_REQUESTED** [review]
agreed to delete all jobs which used only in the test repo

**Comment by lyubenkov** [comment]
> agreed to delete all jobs which used only in the test repo

done https://github.com/neonlabsorg/proxy-model.py/actions/runs/8671991134

File: .github/workflows/deploy.py (hunk 1 above)
Comment (lyubenkov) [reply .github/workflows/deploy.py:487]: Can you please specify one example of it, as it seems that the old logic was located in `github_api_client` file, which is deleted already.

Comment (kristinaNikolaevaa) [reply .github/workflows/deploy.py:487]: I missed it, sorry

**Review by kristinaNikolaevaa — APPROVED** [review]

**Pull request merged**